app/src/pages/App.js

```diff
@@ -2,16 +2,20 @@ import { Header } from './header';
 import { Movies } from './movies';
 import { Footer } from './footer';
 import { ErrorBoundary } from '@globalComponents';
+import { Provider } from 'react-redux';
 import { useState } from 'react';
+import { store } from '@store/store';
 
 function App() {
     const [movieId, setMovieId] = useState(null);
 
     return (
         <ErrorBoundary>
-            <Header showSearchPanel={setMovieId} movieId={movieId} />
-            <Movies showMovieDetails={setMovieId} />
-            <Footer />
+            <Provider store={store}>
+                <Header showSearchPanel={setMovieId} movieId={movieId} />
+                <Movies showMovieDetails={setMovieId} />
+                <Footer />

```

Comment:
will talk about it on the call

Reply:
Done: added `AppContainer` and allocated there `ErrorBoundary`, `Provider`